## Parameters
plot_path: "dataset/drl/m4/latest_training"
plot_flag: True
plot_title: "ocs2wb_timeHorizon3"
plot_window_timestep: 20
plot_window_episode: 20

save_flag: True

### NUA TODO: For multi-data plotting. The testing oar data needs to be linked with testing log!!!
# data_folder: ""
data_folder: "dataset/drl/m4/latest_training"

# data_names: ["dataset/drl/benchmark/oar_data_testing_jackalJaco_0_ocs2wb.csv"]
plot_rewards: True
action_sequence: [0]
observation_sequence: [0]